portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,71 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/* Default Page */
+
+/* The navbar container */
+.navi {
+    overflow: hidden;
+    background-color: aliceblue;
+    height: 70px;
+}
+
+/* Navbar links */
+#linkedin{
+    border-radius: 25%;
+}
+
+.navi a {
+    float: right;
+    display: block;
+    color: steelblue;
+    padding:  20px 30px;
+    text-align: center;
+    line-height: 30px;
+    font-size: 20px;
+    text-decoration: none;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+/* Links - change color on hover */
+.navi a:hover {
+    background-color: snow;
+    color: black;
+}
+
+body.default{
+     background-color: slategray;
+}
+
+#profile {
+    border-radius: 50%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 120px;
+}
+
+.header {
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+    color: white;
```

Comment:
The main page looks amazing, but because the other pages have a white background, they end up being written in invisible ink!

Which reminds me of the old web design joke: "black text on white background is readable; white text on black background is stylish, so if you want it to be readable AND stylish, go with black text on black background".

Reply:
I was so confused by this! I was like why are these links not working, even though I see the pages in the project? 😆
I guess they're just not ready yet. Invisible ink is the new "under construction" sign.